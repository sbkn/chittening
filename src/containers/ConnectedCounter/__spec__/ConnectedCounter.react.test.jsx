import React from "react";
import renderer from "react-test-renderer";
import {ConnectedCounter} from "../ConnectedCounter.react.jsx";
import {mapStateToProps} from "../ConnectedCounter.react.jsx";

describe("ConnectedCounter", () => {

	it("should render correctly", () => {

		const tree = renderer.create(
			<ConnectedCounter>Hello World!</ConnectedCounter>
		).toJSON();

		expect(tree).toMatchSnapshot();
	});

	it("mapStateToProps should map state correctly ", () => {

		const mockState = {
			counter: {}
		};

		expect(mapStateToProps(mockState)).toEqual({count: undefined});
	});
});