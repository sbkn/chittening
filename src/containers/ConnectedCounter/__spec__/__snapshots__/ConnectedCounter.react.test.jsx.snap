// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConnectedCounter should render correctly 1`] = `
<div>
  Button clicks: 
  <div />
</div>
`;
